code/modules/mob/living/simple_animal/hostile/human/skeleton.dm

```diff
@@ -48,7 +48,6 @@
 	deathmessage = "collapses into a pile of bones, its gear clanging as it hits the ground!"
 	loot = list(/obj/effect/decal/remains/human,
 				/obj/item/clothing/suit/armor/witchhunter,
-				/obj/item/clothing/head/witchunter,
```

Comment:
Why are we leaving in the witchhunter amour but not that hat. The srm uses the amour.

Reply:
i really hope they don't

Reply:
also im not done